select2/select2.d.ts

```diff
@@ -78,7 +78,7 @@ interface Select2Options {
     */
     templateSelection?: (object: Select2SelectionObject) => any;
     templateResult?: (object: Select2SelectionObject) => any;
-	language?: string;
+    language?: any;
```

Comment:
It can also be an object like this:
`{
    searching: function (params) {
      return 'Searching…';
    }
  }`

Should it look like this instead:
`string | string[] | {}`
?